profile/README.md

```diff
@@ -9,18 +9,19 @@
 </br>
 <h1 align="center">Welcome to the Nillion Network</h1>
 
-
 Our goal is to build humanity's first blind computer. Nillion is a secure
 computation network that decentralizes trust for high value data in the same way
 that blockchains decentralized transactions.
 
 Get started with Nillion in one of the following links:
-* <a href="https://nillion.com">Website</a>
-* <a href="https://docs.nillion.com">Documentation</a>
-* <a href="https://twitter.com/nillionnetwork">X (Twitter)</a>
-* <a href="https://warpcast.com/nillion">Warpcast</a>
-* <a href="https://www.linkedin.com/company/nillionnetwork/mycompany">LinkedIn</a>
-* <a href="https://discord.com/invite/nillionnetwork">Discord</a>: Our official Discord server.
-* <a href="https://github.com/orgs/NillionNetwork/discussions?discussions_q=">GitHub Discussions</a>: This forum is designed to be a central hub for technical discussions related to Nillion, where builders can share ideas, ask questions, showcase their projects, and stay updated on the latest news and announcements from the Nillion team.
-* <a href="https://www.nillhub.com/">NillHub</a>: our community discussion forum. This is a place where all Nills from around the globe can share, learn, and create together, and for anyone whose native language isn't English, there are sub-categories for languages with their own communities too.
-* <a href="https://t.me/Multipartycomputation">MPC 👀</a>: A broader community Telegram group for discussing MPC developments, papers, and implementation examples.
+
```

Comment:
I think this leaves some space which I personally don't like but if you do I don't mind.

Reply:
oops my default vscode formatting updated the spacing to this when I save the file. I'll revert to only make the one line change and PR